src/app/workout/hooks/goals.hook.tsx

```
@@ -0,0 +1,40 @@
+    const handleEdit = (row: any) => () => {
+        const goal = context.goals.find(d => d.id === row.id)
+        const parsedGoal = {
+            ...goal,
+            beginDate: goal?.beginDate.toISOString().split('T')[0],
+            endDate: goal?.endDate.toISOString().split('T')[0]
+        }
+        formContext.setFormState && formContext.setFormState((prev) => ({
+            ...prev,
+            selectedGoal: parsedGoal as any || null,
+            goalFormType: 'edit'
+        }))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)